tests/unit_tests/pyanaconda_tests/modules/payloads/source/test_source_live_os.py

```diff
@@ -259,14 +295,18 @@ def test_setup_install_source_task_invalid_block_dev(self, proxy_getter, os_stat
 
     @patch("pyanaconda.modules.payloads.source.live_os.initialization.blivet.util.mount")
     @patch("pyanaconda.modules.payloads.source.live_os.initialization.stat")
+    @patch("pyanaconda.modules.payloads.source.live_os.initialization.SetUpLiveOSSourceTask"
+           "._calculate_required_space")
     @patch("os.stat")
     @patch_dbus_get_proxy
-    def test_setup_install_source_task_failed_to_mount(self, proxy_getter, os_stat, stat, mount):
+    def test_setup_install_source_task_failed_to_mount(self, proxy_getter, os_stat,
```

Comment:
By the way, the max line length of the Anaconda code is 99. That's not great, but the consistency has a priority. I think you are using a different one. Please, check the configuration of your IDE.